geomstats/geometry/invariant_metric.py

```diff
@@ -107,13 +119,18 @@ def inner_product(self, tangent_vec_a, tangent_vec_b, base_point=None):
 
         Parameters
         ----------
-        tangent_vec_a
-        tangent_vec_b
-        base_point
+        tangent_vec_a : array-like, shape=[n_samples, dimension]
+            First tangent vector at base_point.
+        tangent_vec_b : array-like, shape=[n_samples, dimension]
+            Second tangent vector at base_point.
+        base_point : array-like, shape=[n_samples, dimension], optional
+            default : identity
```

Comment:
Let's put default values within the parameter description (in parenthesis), to follow numpy docstrings' convention closer:
https://numpydoc.readthedocs.io/en/latest/format.html